ONSdigital/eq-author-app
# Validate page to ensure it has at least one answer

### What is the context of this PR?
Validate that a question page has at least one answer

### How to review 
1. Ensure that an error is shown when a question page does not have an answer
1. Ensure the error is removed/added on addition/deletion of an answer
